### DISCUSSION TITLE
Ajout d'un Board pour organisation

### DISCUSSION THREAD

**Discussion by florinesueur** [cnumr/best-practices-wordpress#41] [Announcements]
Hello à tous,

Pour une mise en ligne officielle pour Halloween, organisons-nous !
[https://github.com/orgs/cnumr/projects/5/views/2?layout=board](https://github.com/orgs/cnumr/projects/5/views/2?layout=board)

Si vous n'avez pas accès, merci de me le préciser en réponse.

**Comment by dNicolle** [comment]
Je ne peux pas assigné à qui je veux, je ne me trouve pas par exemple, ni aucunes des personnes qui travaillent sur le sujet.

**Reply by florinesueur** [reply]
Vous n'êtes dans aucune team du CNUMR. Je vais voir pour créer notre team du coup.

**Reply by florinesueur** [reply]
Je reviens vers vous dès que j'ai du nouveau à ce sujet. Désolée, je n'avais pas prévu ça.
